test/utils.py

```diff
@@ -1101,8 +1101,8 @@ def check_contains_dir(container, dirpath):
     return run_command_on_container(container, command) == 'true'
 
 
-def check_contains_file(container, filepath):
-    command = '[ -f "{}" ] && echo true || echo false'.format(filepath)
+def check_contains_file(container, filepath, is_socket=False):
+    command = '[ -{} "{}" ] && echo true || echo false'.format('S' if is_socket else 'f', filepath)
```

Comment:
Maybe it would be better to split this function to two functions: `check_contains_regular_file` and `check_contains_socket_file`?

Reply:
agree